### DISCUSSION TITLE
Update Step 1 of Intro to account for CodeSpaces

### DISCUSSION THREAD

**Discussion by markdav-is** [skills/.github#154] [Ideas]
Step 1 of the [intro skill](https://github.com/skills/introduction-to-github). shows me the following image:
 
![image](https://user-images.githubusercontent.com/311063/215218304-3f59f70c-0792-4626-a2ca-1bdbb264bb5c.png)

...but, when my team went there, they saw the following: 

![image](https://user-images.githubusercontent.com/311063/215217616-da9e262d-14ea-401c-8861-06292e33f2cb.png)

And it made things confusing

**Comment by cmwilson21** [comment]
Thanks for sharing this feedback! Sorry for the confusion. We will be tracking updating these screenshots internally 💛
